inference-engine/ie_bridges/c/src/ie_c_api.cpp

```diff
@@ -1692,3 +1692,63 @@ void ie_blob_free(ie_blob_t **blob) {
         *blob = NULL;
     }
 }
+
+IEStatusCode ie_blob_get_rwmap(const ie_blob_t *blob, ie_blob_map_t *blob_rwmap) {
+    if (blob == nullptr || blob_rwmap == nullptr) {
+        return IEStatusCode::GENERAL_ERROR;
+    }
+
+    try {
+        IE::MemoryBlob::Ptr memory_blob = IE::as<IE::MemoryBlob>(blob->object);
+        if (memory_blob)
+            blob_rwmap->rwmap = memory_blob->rwmap();
```

Comment:
The `rwmap` will return temporary `LockedMemory` object, which will return mapped pointer and immediately will be destroyed and will call unmap in the destructor.

Reply:
agree. Looks like we need to keep the `LockedMemory` itself somewhere in `blob_rwmap` impl